src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -236,7 +236,7 @@ Currently supported image contrasts include:
 |---------- | ---------------- | ---------------------------------------------------------------------------------------------------- |
 | BOLD      | bold             | Blood-Oxygen-Level Dependent contrast (specialized T2\* weighting)                                   |
 | CBV       | cbv              | Cerebral Blood Volume contrast (specialized T2\* weighting or difference between T1 weighted images) |
-| Phase     | phase            | Phase information associated with magnitude information stored in BOLD contrast                      |
+| Phase     | phase            | Phase information associated with magnitude information stored in BOLD contrast. [Deprecated](../02-common-principles.md#definitions).         |
```

Comment:
Let's specify what it should be replaced with, as well.

Reply:
I'm really looking forward to a later version of the work in #610 so I won't have to edit Markdown tables directly ever again.. But yes, will do.